README.md

```diff
@@ -141,7 +141,7 @@ Requirements: Python 3.9.
   * install Python packages:
 
 ```bash
-pip install .
+pip install -r requirements.txt
```

Comment:
Sorry, but `pip install -r requirements` will only install Python dependencies and not the project itself, so `map-machine` command will not be available from the command line.